frontend/packages/core/src/tab.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { TabProps as MuiTabProps, TabsProps as MuiTabsProps } from "@material-ui/core";
+import { Tab as MuiTab, Tabs as MuiTabs } from "@material-ui/core";
+
+const StyledTab = styled(MuiTab)({
+  minWidth: "111px",
+  height: "46px",
+  padding: "0",
+  color: "rgba(13, 16, 48, 0.6)",
+  borderBottom: "3px solid #E7E7EA",
```

Comment:
it looks a little thicker on figma. i think it should be 4px
```suggestion
  borderBottom: "4px solid #E7E7EA",
```

Reply:
actually nvm, it's correct. on figma it's only the active tab that has the thicker border